spread.yaml

```diff
@@ -8,15 +8,28 @@ environment:
 backends:
   lxd:
     systems: [ubuntu-16.04]
+  linode:
+    key: "$(HOST: echo $SPREAD_LINODE_KEY)"
+    systems:
+      - ubuntu-16.04-64:
```

Comment:
The default linode kernels don't play well with snapd because of the lack of apparmor support, not sure if this apply here but, because you install snapd later, maybe you should use a specific kernel like we do in snapd's suite https://github.com/snapcore/snapd/blob/master/spread.yaml#L51